cgmanifests/generate_cgmanifest.py

```diff
@@ -1,10 +1,28 @@
 #!/usr/bin/env python3
 
+import argparse
+import csv
 import json
 import os
 import re
 import subprocess
 import sys
+from dataclasses import dataclass
+from pathlib import PurePosixPath
+from urllib.parse import urlparse
+
+import requests
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--username", required=True, help="Github username")
```

Comment:
can we update the readme too?
https://github.com/microsoft/onnxruntime/blob/main/cgmanifests/README.md